app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitItemAdapter.java

```diff
@@ -87,35 +86,6 @@ protected void onBindViewHolder(@NonNull HabitViewHolder holder,
                                     @NonNull Habit habit) {
         // set necessary elements of the habit
         holder._habitTitle.setText(habit.getTitle());
-
-
-        setOnClickListenerForHabit(holder, position);
-    }
-
-    /** This method sets the click listener for each habit row
-     * @param holder the ViewHolder that holds each habit
-     * @param position the position of the clicked element
-     */
-    private void setOnClickListenerForHabit(@NonNull HabitViewHolder holder, int position) {
-        holder._habitRows.setOnClickListener(view -> openHabitView(holder, position));
-    }
-
-    /**
-     * This function opens the habit view
-     *
-     * @param holder   This holds the values for the selected habit item
-     * @param position This is the position of the selected habit item
-     */
-    private void openHabitView(HabitViewHolder holder, int position) {
-        // Get the selected habit
-        Habit currentHabit = _snapshots.get(position);
-        // Create a bundle to be passed into the habitViewFragment
```

Comment:
Clicking to view habit has been implemented with RecyclerTouchListener as not doing so caused a certain region to do nothing when clicked